.gitignore

```diff
@@ -1,4 +1,6 @@
+dist
```

Comment:
The built file needs to be accessible for users to download directly from Github.

Reply:
It should be uploaded as a Github release then.

One doesn't keep .exe files in repos either...

Reply:
This pattern needs to be used currently for support for Zenodo and Bower.  While not many people use Bower in new projects, the entire ecosystem is still compatible with it.  It's valuable for legacy projects.  I agree that Bower support should be removed in future, and it should be coordinated with a future Cytoscape v4 release.  For now, Bower support should remain.

Zenodo is important for users in the sciences and academia.  The Cytoscape Consortium is committed to supporting this set of users, and so we must continue to support Zenodo in a sustainable way.

Reply:
Okay, I appreciate the explanation regarding Zenodo. Based on e.g. https://guides.github.com/activities/citable-code/ Zenodo uses GitHub Releases for this – would it be enough to have the built "binary" dist file in the GitHub release for citeability? 

As for Bower, it looks like this library isn't in the Bower index at all, or at least I couldn't find it. https://bower.io/search/ 
Doesn't mean people wouldn't be able to add it by the GitHub repo link, but do you think people really _do_ that?

Reply:
I've reinstated the built file into the repository (under `dist/` as is generally standard), but the commit removing Bower is still there.

After all, it looks as though no one ever even did the Bower registration step mentioned in the readme...

Reply:
You're right, @akx .  Since this repo started with @jfstephe, it seems this extension never supported Bower.  In that case, we don't need to worry about legacy applications.  As long as things work for Zenodo under `/dist`, it should be fine.